### PR TITLE
feat!: introduce Identifiers

### PR DESCRIPTION
**Pull request by antouhou**
## Issue being fixed or feature implemented

Update DPP to v0.16 with binary Identifiers instead of strings

## What was done?

- DPP, dapi-client, and wallet-lib are updated to v0.16
- Methods accept strings and Identifiers
- Introduced ClientApps class

## How Has This Been Tested?

Run the test suite

## Breaking Changes

`client.apps` is an instance of `ClientApps` class.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

**Review by shumkov — DISMISSED** [review]
Looks good!

File: .travis.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ install:
   - npm ci
   - cp .env.example .env
   # Get the latest version of travis-ci-tools
-  - curl -s https://api.github.com/repos/dashevo/travis-ci-tools/releases/latest | jq -r '.tarball_url' | xargs -I {} -n1 wget -O travis-ci-tools.tar.gz -q {}
+  - curl -u shuplenkov:$GITHUB_TOKEN -s https://api.github.com/repos/dashevo/travis-ci-tools/releases/latest | jq -r '.tarball_url' | xargs -I {} -n1 wget -O travis-ci-tools.tar.gz -q {}
```
Comment (shumkov) [inline .travis.yml:36]: Let's move it to `GITHUB_USER` var?

Comment (antouhou) [reply .travis.yml]: Done

**Review by shumkov — APPROVED** [review]
Looks good! 👍

**Pull request merged**